perception/traffic_light_map_based_detector/src/node.cpp

```diff
@@ -401,6 +405,19 @@ void MapBasedDetector::mapCallback(
       all_traffic_lights_ptr_->insert(static_cast<lanelet::ConstLineString3d>(lsp));
     }
   }
+
+  // crosswalk
+  const auto traffic_rules = lanelet::traffic_rules::TrafficRulesFactory::create(
+    lanelet::Locations::Germany, lanelet::Participants::Vehicle);
+  const auto pedestrian_rules = lanelet::traffic_rules::TrafficRulesFactory::create(
+    lanelet::Locations::Germany, lanelet::Participants::Pedestrian);
```

Comment:
Why Germany is used?

Reply:
It seems to be the only one available. https://docs.ros.org/en/noetic/api/lanelet2_traffic_rules/html/doxygen/structlanelet_1_1Locations.html

Reply:
Cloud you add that comment with the link?